src/main/java/mil/dds/anet/beans/EmailDeactivationWarning.java

```diff
@@ -0,0 +1,64 @@
+package mil.dds.anet.beans;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import io.leangen.graphql.annotations.GraphQLIgnore;
+import java.time.Instant;
+import java.util.Objects;
+import mil.dds.anet.views.AbstractAnetBean;
+
+public class EmailDeactivationWarning extends AbstractAnetBean {
+
+  private ForeignObjectHolder<Person> person = new ForeignObjectHolder<>();
```

Comment:
You actually only use the uuid, so just a `String personUuid` would be sufficient.

Reply:
Yep, the whole Person reference is not used anymore (only in tests)